docs/content/guides/recipes/terraform/howto-custom-provider/index.md

```diff
@@ -0,0 +1,86 @@
+---
+type: docs
+title: "How-To: Configure custom Terraform Providers"
+linkTitle: "Configure custom Terraform Providers"
+description: "Learn how to setup your Radius environment to configure custom Terraform Providers and deploy recipes."
+weight: 500
+categories: "How-To"
+aliases  : ["/guides/recipes/terraform/howto-custom-provider"]
+tags: ["recipes", "terraform", "provider"]
+---
+
+This how-to guide will describe how to:
+
+- Configure a custom Terraform Provider and use it in a Terraform recipe.
+In this example we're going to configure a PostgreSQL Terraform provider in a Radius environment and deploy a recipe.    
+Ref: [Documentation for cyrilgdn/postgresql provider](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs)
+
+### Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+
+## Step 1: Define a secretStore resource and store any sensitive information needed for the custom provider. 
+
+Configure a [Radius Secret Store]({{< ref "/guides/author-apps/secrets/overview" >}}) with any sensitive information needed as input configuration for the custom Terraform Provider. Define the namespace for the cluster that will contain your [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/) with the `resource` property. 
+
+> While this example shows a Radius-managed secret store where Radius creates the underlying secrets infrastructure, you can also bring your own existing secrets. Refer to the [secrets documentation]({{< ref "/guides/author-apps/secrets/overview" >}}) for more information.
+
+Create a Bicep file `env.bicep`, import Radius, and  define your resource:
+
+{{< rad file="snippets/env.bicep" embed=true marker="//SECRETSTORE" >}}
+
+> In this example, we're creating a secret with keys `username` and `password` as sensitive inputs required by the Terraform Provider `cyrilgdn/postgresql`.
```

Comment:
Can you also explain what exactly this "sensitive data" is used for? I'm assuming it's to authenticate into the postgresql provider but would be good to clarify and maybe even link to any of their documentation on authentication.

```suggestion
> In this example, we're creating a secret with keys `username` and `password` as sensitive data required by the Terraform Provider `cyrilgdn/postgresql`.
```

Reply:
updated. pls take a look.  The link for official provider documentation is there in the beginning of the page. I'm not sure if we want to copy it over again here. let me know

Reply:
no need to repeat, i think indicating that it's for auth is good enough